pkg/config/config.go

```diff
@@ -23,6 +23,7 @@ const (
 type Install struct {
 	Auto                   bool              `yaml:"auto,omitempty"`
 	Reboot                 bool              `yaml:"reboot,omitempty"`
+	NoFormat               bool              `yaml:"no-format,omitempty"`
```

Comment:
All other are using underscores ... not a biggie if this is like this for another reason

```suggestion
	NoFormat               bool              `yaml:"no_format,omitempty"`
```